cmd/krew/cmd/update.go

```diff
@@ -120,10 +120,14 @@ func ensureIndexUpdated(_ *cobra.Command, _ []string) error {
 		return errors.Wrap(err, "failed to load plugin index after update")
 	}
 
-	installedPlugins, err := installation.ListInstalledPlugins(paths.InstallReceiptsPath())
+	receipts, err := installation.GetInstalledPluginReceipts(paths.InstallReceiptsPath())
 	if err != nil {
 		return errors.Wrap(err, "failed to load installed plugins list after update")
 	}
+	installedPlugins := make(map[string]string)
+	for _, receipt := range receipts {
+		installedPlugins[receipt.Name] = receipt.Spec.Version
+	}
 
 	showUpdatedPlugins(os.Stderr, preUpdateIndex, posUpdateIndex, installedPlugins)
```

Comment:
this part need more comprehensive and careful refactoring. 
I think we need to think heavier refactoring of this part because of

1. plugin name collisions between indexes
2. we ought to show index/ prefix to updated plugins as well.

Maybe let's comment out this feature (showing updated stuff) and let's add a TODO to reactivate in upcoming PRs. Not sure. @corneliusweig thoughts?